app/util/constants.ts

```diff
@@ -5,3 +5,6 @@ export enum SessionStorageKeys {
 export enum FeatureFlags {
   NAMEX_BANNER = 'namex_banner',
 }
+
+export const KeyOfPriorityQueue = 'PriorityQueue'
+export const StringOfTrue = 'true'
```

Comment:
Is StringOfTrue really needed here? Maybe just use `'true'` directly in app/components/app_header/index.vue.

Reply:
Yep, we can directly use `true`.